EffortlessUI-new/src/Components/Input.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import "../base.css";
+import CodeContainer from "./CodeContainer";
+import Description from "./Description";
+import "./Heading.css";
+import "./Input.css";
+
+
```

Comment:
Please remove extra space. please go through all files and remove extra spaces. This will result in neat and clean code.